### ISSUE TITLE
Project Config support

### ISSUE THREAD

**Issue by michaelroper** [fruitstudios/craft-colorit#10]
Hey team! Just wondering, is there anything on the roadmap for this plugin to support the Craft 3.1 Project Config feature, so that field settings and preset settings can be saved into the project.yaml file and synced between environments?

**Comment by michtio** [comment]
@samhibberd sorry to be demanding but could you priortise this? :)

**Comment by evanwarner** [comment]
Yep, gotta bump the importance of this too. Project config saves the `presetId` chosen on a field, but the preset itself is not saved, so a staging/production environment won't have the preset after a deploy. The current solution is either allowing admin changes on an environment where I otherwise wouldn't, or manually copying the `colorit_presets` database records.

**Comment by sjcallender** [comment]
@samhibberd Any updates on this? Not having presets in the project config is a pain. Hoping the fact that this is a paid plugin encourages some attention to this!

If you don't want to maintain it, let me know and I might venture to put an offer to buy it from you all.

**Comment by proimage** [comment]
This is pretty important; just bit me in the rear on a project.

**Comment by patrikalienus** [comment]
Although I have very little hope of it happening since this thread is from Feb 2019; Please add project config support.

**Comment by patrikalienus** [comment]
I really need to version control this, so as a workaround until project config support is a reality - if ever - I merely copy/paste the entire settings column from each preset in the DB, to a text-file within the repo. It's not exactly elegant and doesn't merge with the next release ofc, but as long as the presets are already defined in the DB, pasting the value is easy enough.

**Comment by Michael-Paragonn** [comment]
Bump!
